arm-eventhub/2017-04-01/swagger/EventHub.json

```diff
@@ -1661,6 +1666,71 @@
       ],
       "description": "Single item in List or Get Event Hub operation"
     },
+    "CaptureDescription": {
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "A value that indicates whether capture description is enabled. "
+        },
+        "encoding": {
+          "type": "string",
+          "description": "Enumerates the possible values for the encoding format of CaptureDescription.",
+          "enum": [
+            "Avro",
+            "AvroDeflate"
+          ],
+          "x-ms-enum": {
+            "name": "EncodingCaptureDescription",
+            "modelAsString": false
```

Comment:
In cases like this, we strongly advise against setting `modelAsString` to `false`: This way, any change to the set of enum values (e.g. removal/renaming of values) becomes a breaking change. Only do this if you are certain that the set will stay like that for a very very long time.
On the other hand, modeling enums as strings makes it robust against or at least usable after any kind of changes in the set.